src/tag/types.js

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import type {OverrideT} from '../helpers/overrides';
+
+export type OverridesT = {
+  Root?: OverrideT<*>,
+  Action?: OverrideT<*>,
+};
+
+export type DefaultPropsT = {
+  overrides: OverridesT,
+  disabled: boolean,
+  isFocused: boolean,
+  isHovered: boolean,
+  isActive: boolean,
+  color: string,
+  onActionClick: (
+    e: SyntheticEvent<HTMLInputElement>,
+    children: React$Node,
+  ) => void,
+};
+
+export type PropsT = {
+  overrides?: OverridesT,
+  disabled?: boolean,
+  isFocused?: boolean,
+  isHovered?: boolean,
+  isActive?: boolean,
```

Comment:
What does it mean for a Tag to be "active"? As a customer of Tag, when would I pass in isFocused?

Reply:
removed